azure-pipelines.yml

```diff
@@ -0,0 +1,159 @@
+resources:
+  repositories:
+  # shared library repository
+  - repository: arcade
+    type: github
+    endpoint: DotNet-Bot GitHub Connection
+    name: dotnet/arcade
+    ref: refs/heads/master
+
+variables:
+  DOTNET_SKIP_FIRST_TIME_EXPERIENCE: true
+
+
+phases:
+
+##   The following is the matrix of test runs that we have. This is
+##   duplicated for each os/arch combination in platform-matrix.yml.
+
+##
+##   Product build       Test build              Test run
+##   (Azure DevOps)      (Azure DevOps)          (helix)
+##
+##   ###########################################################################################
+##
+##   Debug
+##
+##   Checked ----------> Pri0 -----------------> plain runtests
+##           |
+##           \---------> Pri1 -----------------> plain runtests
+##           |                \----------------> jitstress
+##           |                \----------------> gcstress
```

Comment:
Looks like building is about 2 hours for Pri1, and I don't know how long our test runs will be with all of the modes we have. I don't see a reason it would differ drastically from our jenkins ci.
Yes, Pri1 includes Pri0. If there's a reason to we could change this pretty easily.